File: Soma.java
import java.util.Scanner;

public class Soma {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Digite um numero: ");
        int N = sc.nextInt();
        
        int soma = 0;
        
        for (int i = 1; i <= N; i++) {
            soma += i;
        }
        
        System.out.println("A soma de 1 a " + N + " eh igual a " + soma);
        
        sc.close();
    }
}

File: soma.js
//faça a soma de um conjunto de números n

function calcularSoma() {
    const N = parseInt(document.getElementById("numero").value);
    let soma = 0;

    for (let i = 1; i <= N; i++) {
        soma = soma + i;
    }

    document.getElementById("resultado").textContent = `A soma de 1 a ${N} é igual a ${soma}`;
}